docs/conf.py

```diff
@@ -64,6 +64,14 @@
 # so a file named "default.css" will overwrite the builtin "default.css".
 html_static_path = ['_static']
 
+html_context = {
```

Comment:
Since we're using GitHub instead of GitLab I think we should be using the GitHub config variables.

https://docs.readthedocs.io/en/stable/guides/vcs.html